resources/keywords/suite_and_test_teardown.resource

```diff
@@ -0,0 +1,47 @@
+*** Settings ***
+Library     SeleniumLibrary
+Library     BuiltIn
+Library     String
+Library     OpenShiftLibrary
+Library     ../../utils/scripts/dbaas_utils.py
+Library     ../../utils/scripts/util.py
+Library    OpenShiftCLI
+Resource    ../object_repo/login_obj.resource
+Resource    login.resource
+
+
+*** Keywords ***
+Tear Down The Test Suite
+    Log    Test Suite Test Down
+    Login To OpenShift CLI
+    Remove Existing DBaaSNamespaces
+    Remove Existing DBaaSProviderAccounts
+
+Tear Down The Test
+    Remove Existing DBaaSInstances
+    Logout Of OpenShift CLI
+    Close Browser
+
+Remove Existing DBaaSInstances
+    IF  '${instanceName}' != '${EMPTY}'
+       ${cmd_instances}    Set Variable    oc delete -A DBaaSInstance/${instanceName} -n ${newProject} --wait=true
```

Comment:
'${instanceName}' != '${EMPTY}' Even when DBaaS Instances are provisioned from Admin view, because after provisioning them, we move to developer view, create a new project and try to deploy the DBaaSInstance on the new created Project Topology

So consequently, the final command which will get executed will look something like this:
oc delete -A DBaaSInstance/some-name -n 0000-test-project-xyz  --wait=true

However, the DBaaSInstance actually doesn't belong to the 0000-test-project-xyz namespace, it belongs to the redhat-dbaas-operator namespace, so we will see an error saying resource not found and the DBaaSInstance will not get deleted.